README.md

```diff
@@ -1,3 +1,18 @@
-# Unamed engine
+# Hush Engine
 
-TODO:
+Hush is a 3D game engine currently in development that aims to provide a more streamlined, simple and performat way to make high quality games
```

Comment:
Oops, this was supposed to go on another branch to update the engine's info, but bled out here